po/pt_BR.po

```diff
@@ -0,0 +1,128 @@
+# Portuguese translations for getoverhere package
+# Traduções em português brasileiro para o pacote getoverhere.
+# Copyright (C) 2023 THE getoverhere'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the getoverhere package.
+# Rafael Fontenelle <rafaelff@gnome.org>, 2023.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: getoverhere\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-01-03 12:32-0300\n"
+"PO-Revision-Date: 2023-01-03 14:27-0300\n"
+"Last-Translator: Rafael Fontenelle <rafaelff@gnome.org>\n"
+"Language-Team: Brazilian Portuguese <https://br.gnome.org/traducao>\n"
+"Language: pt_BR\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1)\n"
+"X-Generator: Gtranslator 42.0\n"
+
+#: data/io.github.cleomenezesjr.GetOverHere.desktop.in:3
+msgid "Get Over Here"
+msgstr "Get Over Here"
+
+#: data/io.github.cleomenezesjr.GetOverHere.appdata.xml.in:7
+msgid "No description"
+msgstr "Sem descrição"
+
+#: src/window.ui:44
+msgid "_Endpoint"
+msgstr "_Extremidade"
+
+#: src/window.ui:60
+msgid "Request"
+msgstr "Requisição"
+
+#: src/window.ui:63
+msgid "Check the method that is being used before sending a request"
+msgstr "Verifica o método que está sendo usado antes de enviar uma requisição"
+
+#: src/window.ui:64
+msgid "Method"
+msgstr "Método"
+
+#: src/window.ui:80
+msgid "URL"
+msgstr "URL"
+
+#: src/window.ui:88
+msgid "Parameters"
+msgstr "Parâmetros"
+
+#: src/window.ui:91
+msgid "Type"
+msgstr "Tipo"
+
+#: src/window.ui:125
+msgid "Raw"
+msgstr "Bruto"
+
+#: src/window.ui:135
+msgid "Go to edit parameters"
+msgstr "Ir para editar parâmetros"
+
+#: src/window.ui:151
+msgid "Send Resquest"
+msgstr "Enviar requisição"
+
+#: src/window.ui:172
+msgid "_Cookies"
+msgstr "_Cookies"
+
+#: src/window.ui:189
+msgid "New Cookie"
+msgstr "Novo cookie"
+
+#: src/window.ui:195 src/window.ui:258 src/window.ui:321 src/window.ui:540
+msgid "Add"
+msgstr "Adicionar"
+
+#: src/window.ui:205 src/window.ui:268 src/window.ui:331 src/window.ui:550
+msgid "Key"
+msgstr "Chave"
+
+#: src/window.ui:211 src/window.ui:274 src/window.ui:337 src/window.ui:556
+msgid "Value"
+msgstr "Valor"
+
+#: src/window.ui:235
+msgid "_Headers"
+msgstr "Cabeçal_hos"
+
+#: src/window.ui:252
+msgid "New Header"
+msgstr "Novo cabeçalho"
+
+#: src/window.ui:298
+msgid "Auth"
+msgstr "Auth"
+
+#: src/window.ui:315
+msgid "New Auth"
+msgstr "Nova autenticação"
+
+#: src/window.ui:404
+msgid "Loading…"
+msgstr "Carregando…"
+
+#: src/window.ui:436 src/window.ui:478 src/window.ui:511
+msgid "Back"
+msgstr "Voltar"
+
+#: src/window.ui:534
+msgid "New Parameter"
+msgstr "Novo parâmetro"
+
+#: src/window.ui:586
+msgid "_Preferences"
+msgstr "_Preferências"
+
+#: src/window.ui:590
+msgid "_Keyboard Shortcuts"
+msgstr "Atalhos de _teclado"
```

Comment:
Again, another abandoned `_`?

Reply:
It's for mnemonics.

Reply:
Yeah. As mentioned before, I set "t" as mnemonic key.

This particular translation is defaul in all GNOME core, circle or Extra Apps under the Brazilian Portuguese translation team of GTP (which I'm the coordinator), so I used the same translation.